docs/DTW_MP_trial.ipynb

```diff
@@ -0,0 +1,810 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #20.                    if bj &gt;= 0 and ρB[i, bj] &lt; ρA[i, aj]:  # ------------ update
</pre><p>Should this be <code>if bj &lt; k and ρB[i, bj] &lt; ρA[i, aj]]:</code>?</p><p> </p><p>I wonder if we can simply re-use <code>core._merge_topk_PI</code> ? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/733/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
It should works.